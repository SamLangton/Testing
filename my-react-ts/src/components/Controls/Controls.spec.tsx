/*
	Snapshot tests for "./Controls.tsx"
*/
import * as React from 'react';
import Controls from './Controls';
import * as renderer from 'react-test-renderer';;

describe('./Controls', () => {
	it('should render as expected', () =>{
		const component = renderer.create(
			<Controls raceInProgress={false}
			raceStartPosition={true}
			/>
		);
		let tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});