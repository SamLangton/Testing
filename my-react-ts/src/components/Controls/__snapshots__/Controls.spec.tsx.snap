// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`./Controls should render as expected 1`] = `
<div
  className="Controls"
>
  <button
    disabled={false}
    onClick={undefined}
    type="button"
  >
    Start
  </button>
  <button
    disabled={true}
    onClick={undefined}
    type="button"
  >
    Reset
  </button>
</div>
`;
